android/app/build.gradle

```diff
@@ -21,6 +21,11 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
+def appMinSdkVersion = localProperties.getProperty('flutter.minSdkVersion').toInteger()
```

Comment:
`localProperties.getProperty('flutter.minSdkVersion')` is causing an NPE on my machine

Reply:
Have you tried adding flutter.minSdkVersion=21 in your local.properties?